packages/lesswrong/components/votes/lwReactions/ReactionQuotesHoverInfo.tsx

```diff
@@ -86,14 +87,11 @@ const ReactionQuotesHoverInfo = ({react, voteProps, commentItemRef, classes}:{
   const { toggleReaction } = useNamesAttachedReactionsVoting(voteProps);
 
   function handleHoverQuote (quote: string) {
-    clearHighlights(commentItemRef)
-    markHighlights(allQuotes, dimHighlightClassName, commentItemRef)
-    markHighlights([quote], highlightSelectorClassName, commentItemRef)
+    // TODO
   }
 
   function handleLeaveQuote () {
-    clearHighlights(commentItemRef)
-    markHighlights(allQuotes, highlightSelectorClassName, commentItemRef)
+    // TODO
```

Comment:
Do these need to be handled in this PR?  If not, more context on what the TODO actually is would be helpful